nupic/research/frameworks/vernon/mixins/__init__.py

```diff
@@ -42,6 +43,7 @@
 from .maxup import MaxupPerSample, MaxupStandard
 from .multi_cycle_lr import MultiCycleLR
 from .oml import OnlineMetaLearning
+from .permuted_mnist_task_indices import *
```

Comment:
This is the same, since in the permuted_mnist_task_indices.py file, I defined `*` to be

```
__all__ = [
    "PermutedMNISTTaskIndices",
]
```

so `import *` will just import the class.

Reply:
ah ok got it :+1: